frontend/src/main/java/puf/frisbee/frontend/view/GameView.java

```diff
@@ -10,6 +10,7 @@
 import javafx.scene.layout.StackPane;
 import puf.frisbee.frontend.core.ViewHandler;
 import puf.frisbee.frontend.viewmodel.GameViewModel;
+import puf.frisbee.frontend.viewmodel.StartViewModel;
```

Comment:
Der Import wird in dem File nicht genutzt und kann glaub ich raus?

Reply:
Kann hier raus, stammt aus einer zwischenzeitlichen Lösung. Begründung siehe oben.